plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -835,84 +920,83 @@ fu_genesys_usbhub_device_setup(FuDevice *device, GError **error)
 		self->flash_sector_size = sector_size;
 
 	/* setup firmware parameters */
-	switch (self->isp_model) {
-	case ISP_MODEL_HUB_GL3523:
+	switch (self->chip.model) {
+	case ISP_MODEL_HUB_GL3523: {
+		gint var = 0;
+		guint8 bonding = 0;
```

Comment:
may be we can use a single variable, instead of two. keep `bonding` and remove `var`.